.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+name: Release
+
+on:
+  workflow_run:
+    workflows: ["Validate & Build"]
```

Comment:
Does this not mean we'll end up publishing `latest` even for PR builds? I suspect we really want `latest` to just reflect `master`.

Reply:
My understanding is that this workflow will only run on `main`, though I'm not completely sure.

Reply:
Essentially what this is trying to do is to run:
> on main, when "Validate & Build" succeeds